docs/source/start.rst

```diff
@@ -2,9 +2,7 @@
 Start your run
 ==============
 
-Make sure a rocoto module is loaded: ``module load rocoto``
-
-If needed check for available rocoto modules on machine: ``module avail rocoto`` or ``module spider rocoto``
+Make sure a rocoto module is loaded. The easiest way to do this is ``source dev/ush/gw_setup.sh``.
```

Comment:
In `setup.rst` there is a warning and note about sourcing `gw_setup.sh`. Should those be replicated here as well?

```suggestion
Make sure a rocoto module is loaded. The easiest way to do this is ``source dev/ush/gw_setup.sh``.

.. warning::
   Sourcing gw_setup.sh will wipe your existing lmod environment

.. note::
   Bash shell is required to source gw_setup.sh

```